source/blog/2016-12-19-last-week-in-manageiq-armor-like-tenfold-regions-hashes.html.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Last Week in ManageIQ: Armor like Tenfold Regions Hashes"
+date: 2016-12-16
```

Comment:
Change the date to 12-19 (this week, since we're writing about last week :smile: ) 

Great job!!